docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+```bash
+ACK7dO9qpsghIme-wvfI
+```
 {{% alert title="Tip" color="success" %}}
-Alternatively, you can download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+You'll use the extract credentials for subsequent authentication in deployed containers.
 {{% /alert %}}
 
-## Install K8ssandra
+### Verify cluster status
 
-The K8ssandra helm charts make installation a snap. You can override chart configurations during installation as necessary if you're an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
+```bash
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
 
-K8ssandra 1.3.0 can install the following versions of Apache Cassandra:
+**Output plus nodetool example:**
 
-* 3.11.7
-* 3.11.8
-* 3.11.9
-* 3.11.10
-* 4.0.0 (default)
+```bash
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@demo-dc1-default-sts-0:/$ nodetool -u demo-superuser -pw ACK7dO9qpsghIme-wvfI status
+Datacenter: dc1
+===============
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address     Load       Tokens  Owns (effective)  Host ID                               Rack
+UN  10.244.4.4  96.71 KiB  16      100.0%            bbdb6a9a-063b-4565-9704-f4caa6fd80f1  default
+UN  10.244.2.5  96.72 KiB  16      100.0%            61dfa8cc-2a8b-4e8f-ae82-01c51833e0ba  default
+UN  10.244.3.4  96.63 KiB  16      100.0%            369aa179-d96e-4f21-a893-f6e6dc84b396  default
+```
 
-{{% alert title="Important" color="warning" %}}
-K8ssandra comes out of the box with a set of [default values](https://github.com/k8ssandra/k8ssandra/blob/main/charts/k8ssandra/values.yaml) tailored to getting up and running quickly.  Those defaults are intended to be a great starting point for smaller-scale local development but are **not** intended for production deployments.
+{{% alert title="Tip" color="success" %}}
+All nodes should have the status UN, which stands for "Up Normal".
 {{% /alert %}}
 
-To install a single node K8ssandra cluster:
-
-1. Copy the following YAML to a file named `k8ssandra.yaml`:
-
-    ```yaml
-    cassandra:
-      version: "3.11.10"
-      cassandraLibDirVolume:
-        storageClass: local-path
-        size: 5Gi
-      allowMultipleNodesPerWorker: true
-      heap:
-       size: 1G
-       newGenSize: 1G
-      resources:
-        requests:
-          cpu: 1000m
-          memory: 2Gi
-        limits:
-          cpu: 1000m
-          memory: 2Gi
-      datacenters:
-      - name: dc1
-        size: 1
-        racks:
-        - name: default
-    kube-prometheus-stack:
-      grafana:
-        adminUser: admin
-        adminPassword: admin123
-    stargate:
-      enabled: true
-      replicas: 1
-      heapMB: 256
-      cpuReqMillicores: 200
-      cpuLimMillicores: 1000
-    ```
+### Test a few operations
```

Comment:
I'm not sure what you'd prefer here, but in the Azure install issue I streamlined my examples a bit to be more copy and paste friendly (after I got smart and had to do it for the 20th time 😉 

Take a look at https://github.com/k8ssandra/k8ssandra/issues/1281

I use a series of commands like:

```
% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "CREATE TABLE test.users (email text primary key, name text, state text);"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('john@gamil.com', 'John Smith', 'NC');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('joe@gamil.com', 'Joe Jones', 'VA');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "select * from test.users;"

 email          | name          | state
----------------+---------------+-------
 john@gamil.com |    John Smith |    NC
  joe@gamil.com |     Joe Jones |    VA
   sue@help.com |       Sue Sas |    CA
    tom@yes.com | Tom and Jerry |    NV

(4 rows)
```

They could all be put in separate blocks here to be copy and pasted easily.

Reply:
Great idea - easier for user.  Done in the next commit.